README.md

```diff
@@ -31,21 +31,44 @@ The packaging rules (`pip_import`, etc.) are less stable. We may make breaking
 changes as they evolve. There are no guarantees for rules underneath the
 `experimental/` directory.
 
+See the [How to contribute](CONTRIBUTING.md) page for information on our
+devlopment workflow.
+
 ## Getting started
 
 To import rules_python in your project, you first need to add it to your
 `WORKSPACE` file. If you are using the [Bazel
-Federation](https://github.com/bazelbuild/bazel-federation), you will want to
-copy the boilerplate in the rules_python release's notes, under the "WORKSPACE
-setup" heading. This will look something like the following:
+Federation](https://github.com/bazelbuild/bazel-federation), you just need to
+[import the Federation](https://github.com/bazelbuild/bazel-federation#example-workspace)
+and call the rules_python setup methods:
 
 ```python
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "bazel_federation",
+    url = "https://github.com/bazelbuild/bazel_federation/releases/download/0.0.1/bazel_federation-0.0.1.tar.gz",
```

Comment:
Incorrect URL: the repo is bazel-federation, not bazel_federation (but the underscore in the basename is actually correct).

I noticed that the federation's release notes contain the same error :(

Reply:
This is really, really confusing. When I copy paste that URL into my browser, it serves me a page saying "Not found". When I edit the underscore in the repo name (not the artifact) into a hyphen, it downloads the artifact but redirects me to the original URL with the incorrect underscore.

Reply:
Pushed the 1-character change.

Reply:
I had the same thing - apparently Chrome replaces the address of a successfully downloaded file with the previous one.

Reply:
Ok, glad it's not a bad github redirect.